src/pages/index.tsx

```diff
@@ -18,11 +21,44 @@ const IndexPage: FunctionComponent = () => {
       />
       <main className="bg-gray-50 dark:bg-gray-900 sm:-my-5 -my-3 -mx-5 p-5">
         <div className="max-w-screen-xl mx-auto">
-          <Home />
+          <Home sections={sections} />
         </div>
       </main>
     </>
   )
 }
 
 export default IndexPage
+
+const digitalGardeningQuery = groq`
```

Comment:
nest step: this query could be organized outside of this file and imported so you'd end up with a library of queries for pulling in specifically shaped data for presentation. It might make sense to pair shaped data like this to a named component (which might be a page?) as well